gufe/components/proteincomponent.py

```diff
@@ -66,15 +72,18 @@ class ProteinComponent(ExplicitMoleculeComponent):
     name : str, optional
        of the protein, by default ""
     """
+    def __init__(self, rdkit: RDKitMol, name=""):
+        if not all(a.GetMonomerInfo() is not None for a in rdkit.GetAtoms()):
+            raise TypeError("Not all atoms in input have MonomerInfo defined.  "
+                            "Consider loading via MolFromPDBFile or similar.")
```

Comment:
```suggestion
                            "Consider loading via rdkit.Chem.MolFromPDBFile or similar.")
```
Just so that it makes sense to folks that don't know rdkit.